### DISCUSSION TITLE
Place elements to default locations in Skin Editor

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25381] [Ideas]
So I made a large discussion post earlier, but I was informed most of it was just duplicates. However, there were a couple of ideas that weren't. I will now go into more depth on one of them in this post. I was told to make separate discussions for each, so this is part 1.

Element Placement: Make it so when an element is added, it appears where the default of that element would usually go

- Makes precise additions easier
- Makes sizing elements up or down faster & easier
- Overall just streamlines the process a bit more

This is how it works in Stable, you simply put an element into the file and they appear where the default would be. I think it would be a welcome addition to Lazer for the skinning community.

**Comment by peppy** [comment]
Not against this, but will require some thought into how the defaults are applied (the current method won't work well with this proposal). I can see how we could make it work though.
